devops/azure-pipelines.yml

```diff
@@ -0,0 +1,48 @@
+# This pipeline relies on the analysis.yml template located in the
+# repository resource descibed below
+resources:
+  repositories:
+  - repository: templates
+    type: github
+    endpoint: Team1-Project3
+    name: 1053-August-Duet-Project-Registry/project-registry-devops
+    ref: refs/heads/feature-templates
```

Comment:
consider whether it makes sense for commits to that repo (on certain paths/branches?) to trigger this one or not.

Reply:
I prefer not to have commits from the devops repo trigger the build.  I figure the devops folks can trigger it manually when they wish.